src/main/java/com/redhat/cloud/policies/app/model/validation/ActionValidator.java

```diff
@@ -33,7 +33,7 @@ public boolean isValid(String input, ConstraintValidatorContext context) {
             return true;
         }
         for (String action : input.split(";")) {
-            String a = action.toLowerCase().strip();
```

Comment:
I'm wondering how we got into that bug if this does a toLowerCase here... 
Do we have other means of updating that field?

Reply:
Ah, this is only validating. The stored data is something else.